pext/ui_qt5.py

```diff
@@ -1572,7 +1572,7 @@ def on_press(self, key) -> bool:
         return True
 
     def on_release(self, key) -> bool:
-        """Executed when a key is released."""
+        """Execute when a key is released."""
```

Comment:
```suggestion
        """Remove key from pressed keys list on release."""
```

Reply:
Fixed in the commit https://github.com/airvzxf/Pext/commit/f765d42edac30a6f25f3e53c0791c76b7c61e31f.